pkg/config/security/security.go

```diff
@@ -108,28 +113,46 @@ func CheckEmptyValues(key string, values []string) error {
 func CheckValidPathTemplate(key string, paths []string) error {
```

Comment:
nit: assuming this is duplicating envoy validation logic, it might be nice to link to it here?